resources/views/checkout/steps/agreements.blade.php

```diff
@@ -21,7 +21,7 @@
     </x-rapidez::slideover>
 
     @if ($agreement->mode == 'AUTO')
-        <label class="text-gray-700 cursor-pointer underline hover:no-underline" for="{{ $agreement->agreement_id }}_agreement">
+        <label class="text-neutral cursor-pointer underline hover:no-underline" for="{{ $agreement->agreement_id }}_agreement">
```

Comment:
text-neutral shouldn't be necessary since the body text is already set on the <body tag

Reply:
Correct, I have removed this.